index.js

```diff
@@ -40,7 +40,7 @@ const net = resolvePath('./mock/empty.js');
 const os = resolvePath('os-browserify/browser.js');
 const path = resolvePath('path-browserify');
 const punycode = resolvePath('punycode/');
-const _process = resolvePath('process/browser.js');
+const _process = resolvePath('./proxy/process').replace('.js', '');
```

Comment:
What is the reason for the `replace`? It breaks this package when used in a projected which defined `"type": "module"` in its `package.json` 👀